src/app/pages/apps/components/installed-apps/app-bulk-update/app-bulk-update.component.ts

```diff
@@ -162,8 +162,7 @@ export class AppBulkUpdateComponent {
   private setInitialValues(): void {
     this.apps.forEach((app) => {
       this.bulkItems.set(app.name, { state: BulkListItemState.Initial, item: app });
-      const [, latestVersion] = app.metadata.app_version.split('_');
-      this.form.addControl(app.name, this.formBuilder.control<string>(latestVersion));
```

Comment:
The split here will always return undefined for `[1]` as there is never a `_` in app_version.